### PR TITLE
Todo component

### PR DESCRIPTION
**Pull request by RemiixSp**

### CODE REVIEW

File: src/ui/style/components/_all.scss [hunk 1]
```diff
@@ -1 +1,2 @@
 @import './header';
```
Comment (Agtevoase) [inline src/ui/style/components/_all.scss:1]: use @use instead of @import

**Review by Agtevoase — CHANGES_REQUESTED** [review]
1. Don't use png/jpg/other raster format for icons. Use SVG instead.
2. Change all styles used in components to modular ones

File: src/ui/pages/home.tsx [hunk 2]
```diff
@@ -1,9 +1,48 @@
 import React from 'react';
 import Header from '../components/header';
+import TodoBlock from '../components/todoBlock';
 const Home = () => {
+  let a = 0;
```
[the remark this answers is not in the document]
Comment (RemiixSp) [reply src/ui/pages/home.tsx:6]: In future it'll be something like array of todos length, but now it's just for testing whether conditional rendering work properly

File: src/ui/style/pages/home.module.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  .home {
+    .todoBlock {
+      // border: solid 1px black;
+      width: 45%;
+      min-height: 200px;
+      margin-top: 30px;
+
+      &__addContainer {
+        height: 300px;
+        h2 {
```
Comment (Agtevoase) [inline src/ui/style/pages/home.module.scss:14]: It's a preference, but it would be nice if your css rules on one level were surrounded by blank lines

**Review by Agtevoase — CHANGES_REQUESTED** [review]

**Review by Agtevoase — APPROVED** [review]

**Pull request merged**